evap/staff/tests/test_views.py

```diff
@@ -874,6 +874,24 @@ def test_form(self):
         self.assertIn("uiae", mail.outbox[0].body)
 
 
+class TestSendReviewReminderTemplate(WebTestStaffMode):
+    @classmethod
+    def setUpTestData(cls):
+        cls.manager = make_manager()
+        cls.evaluation = baker.make(Evaluation, course__responsibles=[], state=Evaluation.State.EVALUATED)
```

Comment:
Is `course__responsibles=[]` required here?

Reply:
No, I've removed it